frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/test-expand.scenario.ts

```diff
@@ -0,0 +1,68 @@
+import { execSync } from 'child_process';
+import { clickKebabAction } from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { testPVC, testDeployment } from '../../mocks/expand-test-mocks';
+import { requestSizeValueInput } from '../../views/add-capacity.view';
+import {
+  goToPersistentVolumeClaims,
+  expandButton,
+  createNewPersistentVolumeClaim,
+  expandSizeOption,
+  capacityUnitDropdown,
+} from '../../views/pvc.view';
+import { SIZE_UNITS } from '../../utils/consts';
+import { sendKeys, getPVCRequestedStorage } from '../../utils/helpers';
+
+const expandValue = String(Number(testPVC.size) + 1);
+
+const createDeployment = () =>
+  execSync(`echo '${JSON.stringify(testDeployment)}' | kubectl create -f -`);
+
+const expandPVC = async (value: string, sizeUnit: SIZE_UNITS) => {
+  await clickKebabAction(testPVC.name, 'Expand PVC');
+  sendKeys(requestSizeValueInput, value);
+  await click(capacityUnitDropdown);
+  await click(expandSizeOption(sizeUnit));
+  await click(expandButton);
+};
+
+describe('Tests Expand flow for PVC', () => {
+  beforeAll(async () => {
+    await createNewPersistentVolumeClaim(testPVC, true, createDeployment);
+  });
+
+  beforeEach(async () => {
+    await goToPersistentVolumeClaims();
+  });
+
+  it('Test PVC can be expanded (In MiBs)', async () => {
+    await expandPVC(expandValue, SIZE_UNITS.MI);
+
+    const pvcJSON = JSON.parse(
+      execSync(`kubectl get pvc ${testPVC.name} -n ${testName} -o json`).toString(),
+    );
+    const requestedStorage = getPVCRequestedStorage(pvcJSON);
```

Comment:
I was basically referring to taking out the line `42` and `45` in a single function and using it all the three places.

Reply:
I don't want to take execSync out of a `scenario` file and put it in a `util` file. The reason being `util` file has helpers that are not performing data fetching or commands so that they are general enough to be used in other places. I don't find the logic too repetitive though? However, if you want I can refactor it out into a function in the same file but I am not sure what we will achieve with that.

Reply:
You can take out the code in a function in the same file as you did here - https://github.com/openshift/console/pull/4860/files#diff-59b909148a5c745c187d786955eebf8cR19, Since, the same logic is used at 3 places, I feel its better to take out in a separate function and use it.

Reply:
ack. Not an issue.

Reply:
ack.